tools.go

```diff
@@ -0,0 +1,13 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+//go:build ignore
+
+package main
+
+import (
+	_ "cmd/gen-docs"
+	_ "internal/config/config.go"
+	_ "pkg/contracts/main.go"
+	_ "pkg/readerclient/main.go"
```

Comment:
I don't get the purpose of this file as is. As far as I know, the tools file is supposed to include only the **external** Go binary tools so they are included in the mods file.

Reply:
Hm, I was not aware of this distinction. I used it as a compiler so every go generate could be easily looked up in a single file. Looking it now, I realize I just patched on a solution for another problem online on the code and, since it worked, I carried on. Fixing it now.